Add the is_front template flag using path.matcher service

andybroomfield opened the pull request:
Fix #279

Same method as page manager.
This adds the is_front template flag to localgov alert banner for testing with twig.
Will restore the default homepage class in the default template.

andybroomfield committed:
Add the is_front template flag using path.matcher service

Fix #279

Same method as page manager.
This adds the is_front template flag to localgov alert banner for testing with twig.
Will restore the default homepage class in the default template.

markconroy reviewed (approved):
Looks good, let's merge.

andybroomfield commented:
I think I need to add a test? But then I will merge this.

andybroomfield committed:
Add test for front page class

Checks the extra front page class is displayed on the alert banner
template when on the front page, and not when on other pages.

andybroomfield commented:
Ready for re-review now the test is added.

finnlewis reviewed (approved):
Looks good to me, thanks @andybroomfield
